domains/rdk07.is-a.dev.json

```
@@ -0,0 +1,8 @@
+{
+  "owner": {
+    "email": "dhipakrs07@gmail.com"
```

username is required
```suggestion
     "username": "dhipakkumar",
    "email": "dhipakrs07@gmail.com"
```